<launch>
  
  <node name="plan_gpd_grasps_service" pkg="moveit_gpd_pr2_pick" type="moveit_gpd_pr2_pick_plan_gripper_grasp_service" output = "screen">

    <param name="bound_frame" value="odom_combined" />

    <!-- Size of the bound determining if a grasp is valid -->
    <param name="x_bound" value="1.0" />
    <param name="y_bound" value="1.0" />
    <param name="z_bound" value="2.0" />
    
    <!-- Offset of the bound from the origin of the frame -->
    <param name="x_bound_offset" value="0.0" />
    <param name="y_bound_offset" value="0.0" />
    <param name="z_bound_offset" value="0.29" />

    <!-- Offset of the gripper endeffector link from the detected grasp -->
    <param name="grasp_offset" value="0.0" />

    <!-- grasps older than this value in seconds won't be considered anymore -->
    <param name="grasp_cache_time_threshold" value="5.0" />

    <param name="move_group_arm" value="left_arm" />
    <param name="move_group_hand" value="left_gripper" />
  </node>
</launch>
